rosbag2_storage_sqlite3/include/rosbag2_storage_sqlite3/sqlite_storage.hpp

```diff
@@ -132,6 +132,14 @@ class ROSBAG2_STORAGE_DEFAULT_PLUGINS_PUBLIC SqliteStorage
   std::mutex database_write_mutex_;
 };
 
+enum class StoragePresetProfile
+{
+  Resilient,
+  WriteOptimized,
+};
+
+StoragePresetProfile parse_storage_preset_profile(const std::string & profile_string);
+
```

Comment:
@james-rms I think it would be better to move `enum class StoragePresetProfile` and `parse_storage_preset_profile(..)` function inside  `SqliteStorage` class. 
The rational for that is to avoid pollution of the  `rosbag2_storage_plugins` namespace.
other storage plugins may have their own definition for `StoragePresetProfile` enum.